README.md

```diff
@@ -1,3 +1,137 @@
-# Yieldnest Cross Chain
+# Yieldnest Cross-Chain Contracts
```

Comment:
beyond existing instructions let's  please add a section that explains clearly step by step (step 1, step 2 etc.) what scripts to run with what configurations to deploy a contract to a new chain.

Also please specifcy exactly what manual transactions need to be executed (eg. addPeer) and with which account.

Reply:
We will get this added once the bash scripts are ready.